cmd/minishift/cmd/oc_env.go

```diff
@@ -83,7 +90,12 @@ var ocEnvCmd = &cobra.Command{
 
 		var shellCfg *OcShellConfig
 
-		shellCfg, err = getOcShellConfig(config.InstanceConfig.OcPath, forceShell)
+		// When oc-env --profile PROFILE_NAME is used and PROFILE_NAME is not an active profile
+		if constants.ProfileName != profileActions.GetActiveProfile() {
```

Comment:
Hard to reason what actually is happening...

```
instancePath := ""
if constants.ProfileName != profileActions.GetActiveProfile() {
	instancePath = getOcPath()
} else {
	instancePath = config.InstanceConfig.OcPath
}
shellCfg, err = getOcShellConfig(instancePath, forceShell)
```


or even with a default:
```
instancePath := config.InstanceConfig.OcPath
if constants.ProfileName != profileActions.GetActiveProfile() {
	instancePath = getOcPath()
}
shellCfg, err = getOcShellConfig(instancePath, forceShell)
```

Reply:
>  
```instancePath := config.InstanceConfig.OcPath
if constants.ProfileName != profileActions.GetActiveProfile() {
	instancePath = getOcPath()
}
shellCfg, err = getOcShellConfig(instancePath, forceShell)
```
:+1:

Reply:
I will fix it.